android/java/org/chromium/chrome/browser/crypto_wallet/fragments/onboarding_fragments/RestoreWalletFragment.java

```diff
@@ -37,6 +39,12 @@ public class RestoreWalletFragment extends CryptoOnboardingFragment {
     private CheckBox showRecoveryPhraseCheckbox;
     private CheckBox restoreLegacyWalletCheckbox;
     private boolean isLegacyWalletRestoreEnable;
+    private boolean isOnboarding;
+
+    public RestoreWalletFragment(boolean isOnboarding) {
```

Comment:
we shouldn't create a constructor for argument, instead we should pass arguments. Configuration change would use the default constructor and you may loose the value for argument.